### DISCUSSION TITLE
any validation examples using class-validator?

### DISCUSSION THREAD

**Discussion by Morriz** [MichalLytek/typegraphql-prisma#317] [Q&A]
Hi @MichalLytek, I finally built a PoC using your lib, but have trouble finding out how to extend the generated models with class-validators. Do you have an example of that?

**Comment by MichalLytek** [comment]
You can try to use this:
https://prisma.typegraphql.com/docs/advanced/additional-decorators

But there are issues with updating because of Prisma using shared inputs for scalars.

**Reply by Morriz** [reply]
Tnx. So there is currently not an easy solution to using this lib and extending it's output with validators? I come from using mainly the openapi ecosystem, where the schema is the single source of truth to work from. I feel like we miss out on a lot of the options found there. Let me try once more to bolt on validators using your suggestion.
